firefox-ios/Client/Application/RemoteSettings/Application Services/ASTranslationModelsFetcher.swift

```diff
@@ -45,11 +45,11 @@ final class ASTranslationModelsFetcher: TranslationModelsFetcherProtocol, Sendab
         static let translatorVersion = "3.0"
         static let modelsVersion = "1.0"
         static let translatorName = "bergamot-translator"
+        static let pivotLanguage = "en"
     }
 
-    private let service: RemoteSettingsService
```

Comment:
Turns out we don't need the service. We are using this pattern in a few other places. I will log a ticket to clean the other RS collections where we don't need a service instance.